README.md

```diff
@@ -133,10 +133,18 @@ Each piece of code (change/patch) is verified, to make the change is safe to mer
 
 <details>
 <summary>What is Continuous Deployment?</summary><br><b>
+
+A development strategy used by developers to release software automatically into production where any code commit must pass through an automated testing phase. Only when this is successful is the release considered production worthy. This eliminates any human interaction and should be implemented only after production-ready pipelines have been set with real-time monitoring and reporting of deployed assets. If any issues are detected in production it should be easy to rollback to previous working state. 
+
+For more info please read [here](https://searchitoperations.techtarget.com/definition/continuous-deployment#:~:text=Continuous%20deployment%20is%20a%20strategy,visible%20to%20the%20software's%20users.)
```

Comment:
hmm maybe a site that doesn't require you to sign in?

Reply:
chnaged